swagger/en/schemas/gameModeStats.yml

```diff
@@ -6,6 +6,10 @@ properties:
     type: integer
     description: "Number of enemy players this player damaged that were killed by teammates"
     minimum: 0
+  bestRankPoints [PC]:

```

Comment:
I hate to make you change this again, but these new fields will actually show up empty for xbox and pre-2018-10 pc data now. =(